Sources/HiPMobile/ServiceAccessLayer/ContentApiAccesses/Contracts/IUserRatingApiAccess.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (C) 2017 History in Paderborn App - Universität Paderborn
+//  
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//  
+//      http://www.apache.org/licenses/LICENSE-2.0
+//  
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using PaderbornUniversity.SILab.Hip.Mobile.Shared.ServiceAccessLayer.ContentApiDtos;
+using PaderbornUniversity.SILab.Hip.Mobile.Shared.BusinessLayer.Models;
+
+namespace PaderbornUniversity.SILab.Hip.Mobile.Shared.ServiceAccessLayer.ContentApiAccesses.Contracts
+{
+    public interface IUserRatingApiAccess
+    {
+        Task<UserRatingDto> GetUserRating(Exhibit exhibit);
+        Task<bool> SendUserRating(Exhibit exhibit, int rating);
```

Comment:
Could you please document what the return value means?

Reply:
Have you seen this? GitHub has collapsed this comment because this line was changed.